portfolio/src/main/webapp/blogScript.js

```diff
@@ -1,9 +1,36 @@
 window.addEventListener('DOMContentLoaded', (event) => {
+    getURL();
+    getPost();
     getComments();
 });
 
+const number = parent.document.URL.substring(parent.document.URL.indexOf('tag='), parent.document.URL.length);
+const tagStr = number.substring(4);
+
+function getURL() {
+    const urlElement = document.getElementById('current-url');
+    urlElement.value = '/blog.html?tag=' + tagStr;
+}
+
+function getPost() {
+
```

Comment:
With the functions throughout this and other file, the functions alternate between having a blank line after the function header and the body and not having a blank line.

eg.
```
function getPost() {

  fetch(...
```
vs.
```
function getPost() {
  fetch
```

At Google, we omit the blank space (from what I know) so I'd suggest removing the blank line from your functions throughout this file and others.